src/extension.ts

```diff
@@ -97,14 +96,33 @@ export class Extension {
 
     this._playwrightTest = new PlaywrightTest(this._isUnderTest);
     this._recorder = new Recorder(this._vscode, this._playwrightTest);
-    this._testController = vscode.tests.createTestController('pw.extension.testController', 'Playwright');
-    this._testController.resolveHandler = item => this._resolveChildren(item);
-    this._testTree = new TestTree(vscode, this._testController);
     this._debugHighlight = new DebugHighlight(vscode);
```

Comment:
I don't think we need these (debug highlight + workspace observer) before we get our first config.

I wonder if a better approach would be to install the `**/*playwright*.config.{ts,js,mjs}` watcher and activate the entire extension based on whether we have configs. I.e. keep the extension object as is, but wrap it into something that'll activate and dispose it.